verification/curator-service/ui/src/components/App.tsx

```diff
@@ -290,6 +290,15 @@ function ProfileMenu(props: { user: User }): JSX.Element {
                 >
                     <MenuItem>Report an issue</MenuItem>
                 </a>
+                <a
+                    href="https://github.com/globaldothealth/list#globalhealth-list"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                    data-testid="forkme"
+                    onClick={handleClose}
+                >
+                    <MenuItem>Fork me on github</MenuItem>
```

Comment:
Maybe just "Github" ? People could go there for documentation and to view the code as well. I know it's a common phrase but "fork me" sounds a bit icky to me.

Reply:
Sure, I concur this website might not be viewed by developers familiar with the term so changed to "view source on github"